tests/conftest.py

```diff
@@ -97,6 +101,31 @@ def data_instance():
     return Data(grid)
 
 
+@pytest.fixture
+def dummy_carbon_data():
+    """Creates a dummy carbon data object for use in tests."""
+
+    from virtual_rainforest.core.data import Data
+    from virtual_rainforest.core.grid import Grid
+
+    # Setup the data object with four cells.
+    grid = Grid(cell_nx=4, cell_ny=1)
+    data = Data(grid)
+
+    # Add the required data.
+    data["maom"] = DataArray([2.5, 1.7, 4.5, 0.5], dims=["cell_id"])
```

Comment:
general comment on variable names, could you make them a bit more descriptive?

Reply:
The comment could be more descriptive too

Reply:
I've changed how both `maom` and `lmwc` are named in the `data` object (to `mineral_associated_om` and `low_molecular_weight_c`). I haven't changed them within my module as there they are contained in functions with docstrings that explain the acronyms. I guess the naming of variables within the `data` object is the most important place to be explicit as in there the variables are presented severed from the context they are generally used/defined/documented in.